tests/integration/targets/lookup_hashi_vault/lookup_hashi_vault/tasks/tinyproxy_server.yml

```
@@ -0,0 +1,19 @@
+---
+- name: Install tinyproxy
+  shell: |
+    apt-get update
+    DEBIAN_FRONTEND=noninteractive apt-get --assume-yes install tinyproxy
+  changed_when: false
+
+- name: Configure tinyproxy
+  copy:
+    content: |
+      Port 8001
+      MaxClients 100
+      StartServers 10
+    dest: /tinyproxy.conf
```

```suggestion
    dest: /tmp/tinyproxy.conf
```
Let's try to account for not running in a disposable/non-persisted FS

---

done
But I used `local_temp_dir` instead of /tmp